### PR TITLE
We should point README readers at the new registry v2 project

### PR DESCRIPTION
**Pull request by estesp**
Docker-DCO-1.1-Signed-off-by: Phil Estes estesp@linux.vnet.ibm.com (github: estesp)

### CODE REVIEW

**Comment by estesp** [comment]
@stevvooe I think this makes sense, otherwise new folks are going to start looking in docker-registry for issues, PRs, etc.  Only thing I wasn't sure is if we want a link to "prior registry v1"

File: README.md [hunk 1]
```diff
@@ -235,8 +235,8 @@ Docker platform to broaden its application and utility.
 If you know of another project underway that should be listed here, please help
 us keep this list up-to-date by submitting a PR.
 
-* [Docker Registry](https://github.com/docker/docker-registry): Registry 
-server for Docker (hosting/delivering of repositories and images) 
+* [Docker Registry v2](https://github.com/docker/distribution): Registry
```
Comment (dmp42) [inline README.md]: I don't think we should mention "v2" here.

**Comment by dmp42** [comment]
We can point to the legacy python registry from the distribution repo.
2 cents.

@fredlf @SvenDowideit what do you think?

**Comment by estesp** [comment]
@dmp42 I removed the "v2" from the link title.  Agree that the "v1 registry" project can just be linked from distribution github project.

**Comment by moxiegirl** [comment]
LGTM

**Comment by dmp42** [comment]
LGTM

**Comment by stevvooe** [comment]
LGTM

**Comment by SvenDowideit** [comment]
LGTM

**Pull request merged**